tox.ini

```diff
@@ -6,16 +6,20 @@ passenv = DATABASE_URL
 usedevelop = True
 
 deps =
+    coverage
     flake8
     flake8-config-4catalyzer
     mock
     psycopg2-binary
     pytest
-    pytest-cov
 
 extras =
     full: apispec,jwt
 
 commands =
     flake8 .
-    pytest {posargs:--cov}
+
+    # FIXME: We get screwed up coverage when using pytest-cov because we
+    #  declare ourselves as a pytest plugin.
+    coverage run -m pytest {posargs}
```

Comment:
sadface

Reply:
some weird interaction... i think pytest ends up rewriting the bytecode for everything for some reason. not worth spending more time here since this fix is functionally the same